Neuron/Sections/Transaction/TransactionHistoryViewController.swift

```diff
@@ -20,15 +21,20 @@ class TransactionHistoryViewController: UIViewController, UITableViewDelegate, U
     @IBOutlet var warningView: UIView!
     @IBOutlet weak var warningHeight: NSLayoutConstraint!
 
-    var service: TransactionHistoryService?
+    var presenter: TransactionHistoryPresenter?
     var tokenProfile: TokenProfile?
     var tokenType: TokenType = .erc20Token
     var tokenModel: TokenModel! {
         didSet {
             guard tokenModel != nil else { return }
-            service = TransactionHistoryService.service(with: tokenModel)
+            presenter = TransactionHistoryPresenter(token: tokenModel)
+            presenter?.delegate = self
             Toast.showHUD()
-            loadData()
+            _ = view // load view
```

Comment:
Found a few of this `_ = view` usage across the project. I thinks this trick means we have some issue design/layout the view controllers and views.